test/e2e/v3/generate_test.go

```diff
@@ -244,16 +259,31 @@ func GenerateV3WithKustomizeV2(kbc *utils.TestContext, crdAndWebhookVersion stri
 	ExpectWithOffset(1, err).NotTo(HaveOccurred())
 
 	By("creating API definition")
-	err = kbc.CreateAPI(
-		"--group", kbc.Group,
-		"--version", kbc.Version,
-		"--kind", kbc.Kind,
-		"--namespaced",
-		"--resource",
-		"--controller",
-		"--make=false",
-		"--crd-version", crdAndWebhookVersion,
-	)
+	if isWithDeployImagePlugin == true {
+		err = kbc.CreateAPI(
+			"plugins", "go/v3,deploy-image/v1-alpha",
+			"--group", "example.com",
+			"--version", "v1alpha1",
+			"--kind", "Memcached",
+			"--image", "memcached:1.6.15-alpine",
+			"--namespaced",
+			"--resource",
+			"--controller",
+			"--make=false",
+			"--crd-version", crdAndWebhookVersion,
+		)
+	} else {
+		err = kbc.CreateAPI(
+			"--group", kbc.Group,
+			"--version", kbc.Version,
+			"--kind", kbc.Kind,
+			"--namespaced",
+			"--resource",
+			"--controller",
+			"--make=false",
+			"--crd-version", crdAndWebhookVersion,
```

Comment:
we do not use this Option for the new plugin 
the crd-version is deprecated and we just need ensure it with the default scaffold
```suggestion
```